src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -1437,8 +1458,8 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_conv_bias_folded_batch_nor
                          << m.get_match_root()->get_name();
             auto pattern_map = m.get_pattern_map();
 
-            auto m_bn = std::dynamic_pointer_cast<op::BatchNormInference>(m.get_match_root());
-            auto m_conv = std::dynamic_pointer_cast<op::ConvolutionBias>(m_bn->get_argument(2));
+            auto m_bn = std::static_pointer_cast<op::BatchNormInference>(m.get_match_root());
+            auto m_conv = std::staticc_pointer_cast<op::ConvolutionBias>(m_bn->get_argument(2));
```

Comment:
Just curious, did you try building this? I only ask because you misspelled 'static_pointer_cast' as static`c`_pointer_cast.